backend/fms_core/schema_validators.py

```diff
@@ -122,13 +123,40 @@ def deconstruct(self):
                             "items": {
                                 "type": "object",
                                 "properties": {
-                                    "external_project_id": {"type": "string"},
                                     "PROJECT": {"type": "string"},
                                 },
                                 "required": ["external_project_id", "PROJECT"]
                             },
                             "minItems": 1,
                         },
+                        "fastq_1": {
```

Comment:
Not sure we should validate the files types we get. Not until we are confident we won't get other types.